cube/cube.c

```diff
@@ -2002,6 +2034,7 @@ static VkShaderModule demo_prepare_shader_module(struct demo *demo, const uint32
 
     err = vkCreateShaderModule(demo->device, &moduleCreateInfo, NULL, &module);
     assert(!err);
+    demo_name_object(demo, VK_OBJECT_TYPE_SHADER_MODULE, (uint64_t)module, "%s", name);
```

Comment:
Couldn't this be simplified to pass name as the string directly? For example:
```c
    demo_name_object(demo, VK_OBJECT_TYPE_SHADER_MODULE, (uint64_t)module, name);
```

Reply:
Some code checking tools become angry when you pass a non-literal string as the format argument to a printf-like function.

Reply:
Ahhh okay, I figured that might of been the case. Good to know for sure.